packages/editor/src/store/selectors.js

```diff
@@ -802,6 +802,29 @@ export const isSavingPost = createRegistrySelector( ( select ) => ( state ) => {
 	);
 } );
 
+/**
+ * Returns true if non-post entities are currently being saved, or false otherwise.
+ *
+ * @param {Object} state Global application state.
+ *
+ * @return {boolean} Whether non-post entities is being saved.
+ */
+export const isSavingNonPostEntityChanges = createRegistrySelector(
+	( select ) => ( state ) => {
+		const entitiesBeingSaved = select(
+			coreStore
+		).__experimentalGetEntitiesBeingSaved();
+		const { type, id } = getCurrentPost( state );
+		return some(
+			entitiesBeingSaved,
+			( entityRecord ) =>
+				entityRecord.kind !== 'postType' ||
+				entityRecord.name !== type ||
+				entityRecord.key !== id
+		);
```

Comment:
This is copied almost verbatim from `hasNonPostEntityChanges`. We might want to extract into a shared helper (and call it something like `excludePostEntity`).

Reply:
Hmm, with the `some()`, we'd need to call it `appliesToNonPostEntities` or so. Not sure it's worth it; might leave it inline for now.